src/pages/TermsOfService.js

```diff
@@ -240,4 +241,37 @@ class TermsOfService extends Component {
   }
 }
 
-export default TermsOfService
+class TermsOfServicePage extends Component {
+  render() {
+    return div({ style: styles.page }, [
+      backgroundLogo,
+      div({ style: styles.box }, [
+        div({ style: { color: colors.gray[0], fontWeight: 600 } }, [
+          span({ style: { fontSize: 36 } }, ['TERRA ']),
+          span({ style: { fontSize: 24 } }, ['Terms of Service'])
+        ]),
+        div({ style: { maxHeight: 400, overflowY: 'auto', lineHeight: 1.5, marginTop: '1rem', paddingRight: '1rem' } }, [
+          h(Markdown, {
+            renderers: {
+              heading: (text, level) => {
+                return `<h${level} style="color: ${colors.gray[0]}; margin-bottom: 0">${text}</h${level}>`
+              },
+              paragraph: text => {
+                return `<p style="margin-top: 0">${text}</p>`
+              }
+            }
+          }, [termsOfService])
```

Comment:
There's a fair bit of duplication here. Though from a UX perspective I wonder if the page version should look different, e.g. full-screen, since the focus is on the content rather than the 'Accept' button.